pipelines/DECam/ApPipeWithFakes.yaml

```diff
@@ -11,9 +11,15 @@ imports:
   - location: $AP_PIPE_DIR/pipelines/_ingredients/ApPipeWithFakes.yaml
 
 parameters:
-  apdb_config:  # YOU MUST CONFIGURE THIS BEFORE RUNNING THE PIPELINE
+  coaddName: goodSeeing
+  # APDB config file must be user-specified
+  apdb_config:
+  fakesType: 'fakes_'
+  injected_prefix: 'fakes_'
+  injection_prefix: 'injection_'
 tasks:
   isr:
     class: lsst.ip.isr.IsrTask
     config:
       file: $AP_PIPE_DIR/config/DECam/runIsrWithCrosstalk.py
+      connections.outputExposure: post_isr_image
```

Comment:
Why is this configured here and not in `_ingredients`? Or, barring that, `runIsrWithCrosstalk.py`?